src/components/LinkChooser/types.ts

```diff
@@ -130,10 +130,15 @@ export type SearchInputProps = {
     machineService: MachineService;
     validation: Validation;
     onClear?: () => void;
+    wrapperProps?: {
```

Comment:
is the wrapper needed? In the end it is the component emitting it

Reply:
After design review a checklist item was created since clicking inside the wrapper but not on the input would not open the dropdown, so I had to add additional listeners incase the area around the input was clicked

Reply:
Feel a bit weird, can't we align it like that?
```
    validation: Validation;
    onClear?: () => void;
    onClick?: (event: MouseEvent<HTMLDivElement>) => void;
    onMouseDown?: (event: MouseEvent<HTMLDivElement>) => void;
```